packages/integration-tests/index.js

```diff
@@ -256,6 +256,7 @@ function setProcessEnvironment(stackName, bucketName) {
   process.env.CollectionsTable = `${stackName}-CollectionsTable`;
   process.env.ProvidersTable = `${stackName}-ProvidersTable`;
   process.env.RulesTable = `${stackName}-RulesTable`;
+  process.env.noSNS = true;
```

Comment:
I realize that this eliminates the warning message I was getting when running integration tests, but why not just set the environment variables for the SNS topics so that posting to them will actually succeed?

```
process.env.granule_info_sns_topic_arn = `${stackName}-granule-information-topic`
```